homeassistant/components/zwave_js/update.py

```diff
@@ -143,7 +146,16 @@ def _unsub_firmware_events_and_reset_progress(
 
     async def _async_update(self, _: HomeAssistant | datetime | None = None) -> None:
         """Update the entity."""
-        self._poll_unsub = None
+        if self._poll_unsub:
+            self._poll_unsub()
+            self._poll_unsub = None
+
+        # If hass hasn't started yet, push the next update to the next day so that we
+        # can preserve the offsets we've created between each node
+        if not self.hass.is_running:
```

Comment:
I don't know how to test this, but I am pretty sure it is needed.

Reply:
You added a test for this, right?

Reply:
no I don't have a test for this yet

Reply:
updated a test to handle this